### ISSUE TITLE
Add support for Robovm 2.x

### ISSUE THREAD

**Issue by keesvandieren** [libgdx/gdx-pay#123]
Support the open sources Robovm 2.x

Workaround with current release: exclude org.robovm dependencies, like so:

```
compile("com.badlogicgames.gdxpay:gdx-pay-iosrobovm-apple:${gdxPayVersion}") {
    exclude group: 'org.robovm'
}
```

**Comment by codeulike** [comment]
I'm using RoboVM 2.3.0 with GdxPay 0.10.3, seems to work ok on ios

Here's the ios bit of my build.gradle:
```
project(":ios") {
    apply plugin: "java"
    apply plugin: "robovm"


    dependencies {
        compile project(":core")
        compile "com.mobidevelop.robovm:robovm-rt:$roboVMVersion"
        compile "com.mobidevelop.robovm:robovm-cocoatouch:$roboVMVersion"
        compile "com.badlogicgames.gdx:gdx-backend-robovm:$gdxVersion"
        compile "com.badlogicgames.gdx:gdx-platform:$gdxVersion:natives-ios"
        compile "com.badlogicgames.gdxpay:gdx-pay-client:$gdxPayVersion"
        compile "com.badlogicgames.gdxpay:gdx-pay-iosrobovm-apple:$gdxPayVersion"
    }
}
```

**Comment by keesvandieren** [comment]
Just released 0.11.0 with Robovm 2.3.0
